src/tictacclojure/console_ui.clj

```diff
@@ -0,0 +1,28 @@
+(ns tictacclojure.console-ui)
+
+(defn print-out
+  [message]
+  (println message))
+
+(defn- always-validate
+  [user-input]
+  true)
+
+(defn get-input
+  ([]
+    (get-input "" always-validate ""))
+
+  ([prompt]
+    (get-input prompt always-validate ""))
+
+  ([validate error-message]
+   (get-input "" validate error-message))
+
+  ([prompt validate error-message]
+    (if (not-empty prompt) (print-out prompt))
```

Comment:
I think thins might reveal a slight flaw in how I treat the different types of players (human and computer), but in an effort to try make things easier, the prompt for a new move always happens from the game runner, which is why the human player then uses `get-input` without a prompt.